crates/broker/src/order_picker.rs

```diff
@@ -117,12 +127,30 @@ where
             return Ok(false);
         };
 
-        // is the order expired already?
-        // TODO: Handle lockTimeout separately from timeout.
-
-        let expiration = order.request.offer.biddingStart + order.request.offer.lockTimeout as u64;
+        // Lock expiration is the timestamp before which the order must be filled in order to avoid slashing
+        let lock_expiration =
+            order.request.offer.biddingStart + order.request.offer.lockTimeout as u64;
+        // order expiration is the timestamp after which the order can no longer be filled by anyone.
+        let order_expiration =
+            order.request.offer.biddingStart + order.request.offer.timeout as u64;
 
         let now = now_timestamp();
+        // If order_expiration > lock_expiration the period in-between is when order can be filled
+        // by anyone without staking to partially claim the slashed stake
+        let slashed_unexpired = order.status == OrderStatus::Slashed
+            && lock_expiration <= now
+            && order_expiration > now;
+
```

Comment:
Related to my other comment, but I don't think this works since requests are only slashed once they fully expire. I think we'll need another state that represents "LockExpiredUnfulfilled", and to track the `lockTimeout` for each value